### DISCUSSION TITLE
Limit pause container resources

### DISCUSSION THREAD

**Discussion by nsoubelet** [k3s-io/k3s#3046] [Q&A]
Hello guys,
I am currently using k3s along with gVisor. When I found out that gVisor informs incorrectly the resources pod was limit with, someone from gVisor project replied that it is related to pause container. Please see https://github.com/google/gvisor/issues/190#issuecomment-767203156

So my question is whether there is a way to limit pause container resources in k3s so I am able to propagate correct values to containers in a Pod.

Thanks!

**Comment by brandond** [comment]
The gVisor folks appear to consider this a Kubernetes issue, and not specific to K3s. Have you checked to see if there is an upstream issue for this open yet in the kubernetes/kubernetes project?

**Reply by nsoubelet** [reply]
Hi @brandond 
Thanks, yes in fact this is a Kubernetes issue, but since k3s initiates its own pause container, I thought that maybe there is an easy way for me to solve the problem by limiting it.
The thing is that apparently for Kubernetes upstream devs that is not an issue. See this https://github.com/kubernetes/kubernetes/issues/93055

**Reply by brandond** [reply]
That's not something we do any differently. We're still a Kubernetes distro, and other than a very few things we have modified to make it easier for us to run everything in a single process K3s behaves exactly like upstream.

**Reply by nsoubelet** [reply]
Understood, thanks.
